guide/en/security/authorization.md

```diff
@@ -46,10 +46,10 @@ so an implementation should be provided in dependency container in order for it
 ## Role Based Access Control (RBAC) <span id="rbac"></span>
 
 Role-Based Access Control (RBAC) provides a simple yet powerful centralized access control. Please refer to
-the [Wikipedia](http://en.wikipedia.org/wiki/Role-based_access_control) for details about comparing RBAC
+the [Wikipedia](https://en.wikipedia.org/wiki/Role-based_access_control) for details about comparing RBAC
 with other more traditional access control schemes.
 
-Yii implements a General Hierarchical RBAC, following the [NIST RBAC model](http://csrc.nist.gov/rbac/sandhu-ferraiolo-kuhn-00.pdf).
+Yii implements a General Hierarchical RBAC, following the [NIST RBAC model](https://csrc.nist.gov/rbac/sandhu-ferraiolo-kuhn-00.pdf).
```

Comment:
```suggestion
Yii implements a General Hierarchical RBAC, following the [NIST RBAC model](https://csrc.nist.gov/rbac/sandhu-ferraiolo-kuhn-00.pdf).
```

Link redirected to https://csrc.nist.gov/projects/role-based-access-control — it's actual? If yes, then need overwrite link.

Reply:
fix